.github/workflows/bn_master_commit.yml

```diff
@@ -0,0 +1,91 @@
+name: BabylonNative master branch update
+
+on:
+  repository_dispatch:
+    types: [babylonnative-master-update]
+
+jobs:

```

Comment:
nit: Feels like you could use a template for this instead of needing to copy job definitions: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops

Reply:
im not sure if azure dev ops templates can be used for github actions. We also have different submodule initialization steps in this definitions

Reply:
It seems like pr.yml and bn_master_commit.yml are similar enough to combine-- you could avoid the `initializeSubmodulesMostRecentBabylonNative` with an if on the step using an input to the composite action...

I'm fine with punting the cleanup, just wanted to point it out

Reply:
If it is possible to not duplicate all this that would be great. I hadn't heard of a way to do that with GitHub actions, but maybe the composite thing you mentioned is a solution?

Reply:
it seems like this composite solution is good for defining actions in their own repos. The description doesn't show how to do this for repo specific action reuse. I don't think these actions warrant their own repo

Reply:
definitely agree with the sentiment of not wanting to rewrite the same action content